zerver/lib/actions.py

```diff
@@ -1415,7 +1415,7 @@ def do_add_subscription(user_profile, stream, no_log=False):
         event = dict(type="subscription", op="peer_add",
                      subscriptions=[stream.name],
                      user_email=user_profile.email)
-        send_event(event, user_ids)
+        send_event(event, active_user_ids(user_profile.realm))
```

Comment:
@timabbott In reference to my last [comment](https://github.com/zulip/zulip/pull/749#issuecomment-227593338), I wanted to say that noticed user_ids was `[]`(Hence, user wasn't getting the notice). When I replaced it with `active_user_ids(user_profile.realm)`, `peer_add` event was recieved. I believe it's not the correct way of doing this. Thoughts?
Sorry for that comment